Formula/pinboard-notes-backup.rb

```diff
@@ -0,0 +1,24 @@
+require "language/haskell"
+
+class PinboardNotesBackup < Formula
+  include Language::Haskell::Cabal
+
+  desc "Efficiently back up the notes you’ve saved to Pinboard"
+  homepage "https://github.com/bdesham/pinboard-notes-backup"
+  url "https://github.com/bdesham/pinboard-notes-backup/archive/v1.0.3.tar.gz"
+  sha256 "bc3ab1a8a3d92fcbda86dd8b4756b035be89e1e5b50bdd61f998b67c89243ae3"
+  head "https://github.com/bdesham/pinboard-notes-backup.git"
+
+  depends_on "ghc@8.0" => :build
+  depends_on "cabal-install" => :build
+
+  def install
+    install_cabal_package
+  end
+
+  # A real test would require hard-coding a Pinboard API key here, which would
```

Comment:
You can assert an expected failure of some kind instead, or mock the server (see faas-cli)